package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "api-snippets",
+  "version": "0.0.1",
+  "description": "Mojaloop API specification reusable snippets",
+  "scripts": {
+    "test": "multi-file-swagger test/index.yaml > test/resolved.json && multi-file-swagger -o yaml test/index.yaml > test/resolved.yaml && node test/diffCheck.js",
+    "buildv1.0": "multi-file-swagger -o yaml v1.0/openapi3/index.yaml > docs/fspiop-rest-v1.0-openapi3-snippets.yaml"
+  },
+  "license": "MIT",
```

Comment:
shouldn't be licensed under `Apache-2.0`?

Reply:
I don't see LICENSEE.md file ...

Reply:
Licensing for Mojaloop is something that I don't have full knowledge of. I planned to tackle that in another PR. I'll change it to `Apache-2.0` for now, is that okay?

Reply:
Ok. Talked with Sam. The package will be under Apache 2.0 while the snippets and swagger files are TBD. That license will come after Sam and the team come up with a story to frame this.

Added `LICENSEE.md`